usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -568,43 +565,43 @@ def build_date_format_sql(
+            """
+            Args:
+                is_result_aliased (bool) if true, aliases the parsing result with the given ``col``'s ``dest_name``
+            """
             if col.handling == "cast":
-                retval = (
-                    f"CAST({bronze_table_name}.{col.source} AS {col.delta_type})"
-                    f"{' AS ' + col.dest_name if is_result_aliased else ''}"
-                )
+                retval = f"CAST({bronze_table_name}.{col.source} AS {col.delta_type})"
             elif col.handling == "literal":
                 # Use col.source directly as the value
-                retval = f"{col.source}{' AS ' + col.dest_name if is_result_aliased else ''}"
+                retval = f"{col.source}"
             elif col.handling == "parse_string_datetime_to_date":
                 # These are string fields that actually hold DATES/TIMESTAMPS and need to be cast as dates.
                 # However, they may not be properly parsed when calling CAST(... AS DATE).
-                retval = build_date_format_sql(col, is_casted_to_date=True, is_result_aliased=is_result_aliased)
+                retval = build_date_format_sql(col, is_casted_to_date=True)
             elif col.handling == "string_datetime_remove_timestamp":
                 # These are string fields that actually hold DATES/TIMESTAMPS, but need the non-DATE part discarded,
                 # even though they remain as strings
-                retval = build_date_format_sql(col, is_casted_to_date=False, is_result_aliased=is_result_aliased)
+                retval = build_date_format_sql(col, is_casted_to_date=False)
             elif col.delta_type.upper() == "STRING":
                 # Capitalize and remove leading & trailing whitespace from all string values
-                retval = (
-                    f"ucase(trim({bronze_table_name}.{col.source}))"
-                    f"{' AS ' + col.dest_name if is_result_aliased else ''}"
-                )
+                retval = f"ucase(trim({bronze_table_name}.{col.source}))"
             elif col.delta_type.upper() == "BOOLEAN" and not col.handling == "leave_null":
                 # Unless specified, convert any nulls to false for boolean columns
-                retval = (
-                    f"COALESCE({bronze_table_name}.{col.source}, FALSE)"
-                    f"{' AS ' + col.dest_name if is_result_aliased else ''}"
-                )
+                retval = f"COALESCE({bronze_table_name}.{col.source}, FALSE)"
             else:
-                retval = f"{bronze_table_name}.{col.source}{' AS ' + col.dest_name if is_result_aliased else ''}"
+                retval = f"{bronze_table_name}.{col.source}"
+
+            # Handle scalar transformations if the column requires it
+            if col.scalar_transformation is not None:
+                retval = col.scalar_transformation.format(input=retval)
 
+            retval = f"{retval}{' AS ' + col.dest_name if is_result_aliased else ''}"
```

Comment:
As Clancy said, good change here! 

This would be something good to call out in the Technical Details in the future.